.github/workflows/generate_documentation.yaml

```
@@ -0,0 +1,36 @@
+on:
+  push:
+    branches:
+      - main  
+
+  workflow_dispatch:
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)